packages/plugins/Wallet/src/messages.ts

```diff
@@ -237,3 +239,8 @@ if (import.meta.webpackHot) import.meta.webpackHot.accept()
 export const WalletMessages: { events: PluginMessageEmitter<WalletMessage> } = {
     events: createPluginMessage<WalletMessage>(PLUGIN_ID),
 }
+
+export const MaskMessages = new WebExtensionMessage<MaskEvents>({ domain: 'mask' })
```

Comment:
Please don't move it here. `MaskMessages` is for general purposes.

Reply:
How to use `MaskMessages` in `packages/plugins/FileServices`?

Reply:
please hold on. I need to consider how to handle this

Reply:
@zhouhanseng https://github.com/DimensionDev/Maskbook/pull/5961 as a temporary solution.